### PR TITLE
modules: viss: ldc: added live tuning support

### PR DESCRIPTION
**Pull request by Brandon-Maciel-TI**

### CODE REVIEW

File: src/tiovx_ldc_module.c [hunk 1]
```diff
@@ -496,6 +496,15 @@ vx_status tiovx_ldc_module_init(vx_context context, TIOVXLDCModuleObj *obj, Sens
...
 
+    if((vx_status)VX_SUCCESS == status)
+    {
+        status = itt_register_object(&context,
+                                     &(obj->node),
+                                     NULL,
+                                     NULL,
+                                     LDC);
+    }
+
```
Comment (ravi-rahul) [inline src/tiovx_ldc_module.c:507]: Hi @Brandon-Maciel-TI 

Can we protect it with a macro
and enable it only when needed
via an option in Cmake

Comment (ravi-rahul) [reply src/tiovx_ldc_module.c:507]: you can take below as reference
https://git.ti.com/cgit/edgeai/edgeai-dl-inferer/tree/cmake/common.cmake?h=develop#n6
https://git.ti.com/cgit/edgeai/edgeai-dl-inferer/tree/cmake/common.cmake?h=develop#n50
https://git.ti.com/cgit/edgeai/edgeai-dl-inferer/tree/dl_inferer/src/ti_dl_inferer.cpp?h=develop#n742

**Review by ravi-rahul — CHANGES_REQUESTED** [review]

File: src/tiovx_ldc_module.c (hunk 1 above)
Comment (Brandon-Maciel-TI) [reply src/tiovx_ldc_module.c:509]: done!

**Comment by ravi-rahul** [comment]
Hi @Brandon-Maciel-TI 

This is ready to be merged
Please squash all the commits into a single commit

**Comment by ravi-rahul** [comment]
Hi @Brandon-Maciel-TI 

Please add the appropriate commit message
Please refer to previous commits

**Review by ravi-rahul — APPROVED** [review]

**Pull request merged**